packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraTestUtils.m

```diff
@@ -7,26 +7,78 @@
+  }
+
   id inputMock = OCMClassMock([AVCaptureDeviceInput class]);
   OCMStub([inputMock deviceInputWithDevice:[OCMArg any] error:[OCMArg setTo:nil]])
       .andReturn(inputMock);
 
   id videoSessionMock = OCMClassMock([AVCaptureSession class]);
+  OCMStub([videoSessionMock beginConfiguration])
+      .andDo(^(NSInvocation *invocation){
+      });
+  OCMStub([videoSessionMock commitConfiguration])
+      .andDo(^(NSInvocation *invocation){
+      });
+
   OCMStub([videoSessionMock addInputWithNoConnections:[OCMArg any]]);
   OCMStub([videoSessionMock canSetSessionPreset:[OCMArg any]]).andReturn(YES);
 
   id audioSessionMock = OCMClassMock([AVCaptureSession class]);
   OCMStub([audioSessionMock addInputWithNoConnections:[OCMArg any]]);
   OCMStub([audioSessionMock canSetSessionPreset:[OCMArg any]]).andReturn(YES);
 
-  return [[FLTCam alloc] initWithCameraName:@"camera"
-                           resolutionPreset:@"medium"
-                                enableAudio:true
-                                orientation:UIDeviceOrientationPortrait
-                        videoCaptureSession:videoSessionMock
-                        audioCaptureSession:audioSessionMock
-                        captureSessionQueue:captureSessionQueue
-                                      error:nil];
+  id fltCam = [[FLTCam alloc] initWithCameraName:@"camera"
+                                resolutionPreset:@"medium"
+                                   mediaSettings:mediaSettings
+                          mediaSettingsAVWrapper:mediaSettingsAVWrapper
+                                     orientation:UIDeviceOrientationPortrait
+                             videoCaptureSession:videoSessionMock
+                             audioCaptureSession:audioSessionMock
+                             captureSessionQueue:captureSessionQueue
+                                           error:nil];
+
+  id captureVideoDataOutputMock = [OCMockObject niceMockForClass:[AVCaptureVideoDataOutput class]];
+
+  OCMStub([captureVideoDataOutputMock new]).andReturn(captureVideoDataOutputMock);
+
+  OCMStub([captureVideoDataOutputMock
+              recommendedVideoSettingsForAssetWriterWithOutputFileType:AVFileTypeMPEG4])
+      .andReturn(@{});
+
+  OCMStub([captureVideoDataOutputMock sampleBufferCallbackQueue]).andReturn(captureSessionQueue);
+
+  id videoMock = OCMClassMock([AVAssetWriterInputPixelBufferAdaptor class]);
+  OCMStub([videoMock assetWriterInputPixelBufferAdaptorWithAssetWriterInput:OCMOCK_ANY
+                                                sourcePixelBufferAttributes:OCMOCK_ANY])
+      .andReturn(videoMock);
+
+  id writerInputMock = [OCMockObject niceMockForClass:[AVAssetWriterInput class]];
+
+  OCMStub([writerInputMock assetWriterInputWithMediaType:AVMediaTypeAudio
+                                          outputSettings:[OCMArg any]])
+      .andReturn(writerInputMock);
+
+  OCMStub([writerInputMock assetWriterInputWithMediaType:AVMediaTypeVideo
```

Comment:
I recall that there can be only a single stub for a given method and this will not work as was probably intended, either the first or the second `OCMStub` of `assetWriterInputWithMediaType` is ignored here.